packages/rrweb/src/record/mutation.ts

```diff
@@ -438,10 +438,26 @@ export default class MutationBuffer {
         // text mutation's id was not in the mirror map means the target node has been removed
         .filter((text) => this.mirror.has(text.id)),
       attributes: this.attributes
-        .map((attribute) => ({
-          id: this.mirror.getId(attribute.node),
-          attributes: attribute.attributes,
-        }))
+        .map((attribute) => {
+          const attributes = attribute.attributes;
```

Comment:
Object destructuring is neater here
```suggestion
          const { attributes } = attribute;
```

Reply:
+0
It's actually quite confusing as there is a parent property called `attributes` and each attribute has an `attributes` property, but that should be solved by renaming one of them so no objection to this change.